tsconfig.json

```diff
@@ -11,10 +11,12 @@
     "strict": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "allowJs": true,
+    "declaration": true,
     "typeRoots": ["./src/types", "./node_modules/@types"]
   },
   "include": [
     "src/**/*",
-    "generated/*"
+    "test/**/*"
```

Comment:
Since we now include `test` in our `tsconfig`, see if you can get rid of `tsconfig.eslint.json` and change the `project` key in `.eslintrc.js` back to the standard `tsconfig.json` and still lint successfully.

Reply:
Weird things happened with the rebase I'll remove that.